.github/2.0/cookbooks/Document.md

```diff
@@ -574,12 +574,12 @@ DocumentArray has 3 items:
 
 ### Use `itertools` on `DocumentArray`
 
-As `DocumenArray` is an `Iterable`, you can also use [Python built-in `itertools` module](https://docs.python.org/3/library/itertools.html) on it. This enables advanced "iterator algebra" on the `DocumentArray`.
+As `DocumentArray` is an `Iterable`. You can also use [Python's built-in `itertools` module](https://docs.python.org/3/library/itertools.html) on it. This enables advanced "iterator algebra" on the `DocumentArray`.
```

Comment:
We really should have a dot here? those two sentences are connected right?

Reply:
Ah, my bad. I'll fix

Reply:
Fixed